docs/development.md

```diff
@@ -0,0 +1,74 @@
+# Setting Up a Development Environment
+
+
+## Table of contents
+
+1. [Installing the Zendesk App Tools (ZAT)](#dev-zat)
+2. [Setting up the Python environment](#dev-python)
+3. [Lauching the Flask app](#dev-flask)
+4. [Getting browser connected to the Zendesk instance](#dev-instance)
+
+
+## Installing the Zendesk App Tools (ZAT) <a name='dev-zat'></a>
+ZAT is used for developing Zendesk apps and it bridges communications between your local (or remote) app and your Zendesk product instance.
```

Comment:
would be good to have a reference to the ZAT doc or repository.

Reply:
Good one, link added.